### ISSUE TITLE
Launch files not installed

### ISSUE THREAD

**Issue by CameronDevine** [AprilRobotics/apriltag_ros#36]
When attempting to build `apriltags2_ros` the launch files are not installed into the catkin workspace path. To install I ran,

```bash
cd ~/catkin_ws/src
git clone https://github.com/dmalyuta/apriltags2_ros.git
cd ..
catkin_make
catkin_make install
source install/setup.bash
```

When I attempt to run `roslaunch apriltag2_ros continuous_detection.launch` I get an error message that `continuous_detection.launch` is not a part of `apriltag2_ros`. Using the `find` command I can tell that no launch files from `apriltag2_ros` are in my `~/catkin_ws/install` directory.

**Comment by ceplm** [comment]
The package is apriltags2_ros, you misspelled it.

**Comment by wxmerkt** [comment]
I think this is a valid issue - quick glance at the CMakeLists.txt shows no install directives. Do you mind proposing a pull request? Thanks!

**Comment by ceplm** [comment]
Sorry, I just only installed the package using code above and tried launching it. Since no one commented, I thought the misspelled name could be issue.

**Comment by CameronDevine** [comment]
@wxmerkt I should be able to create a pull request sometime within the next week or so.

**Comment by wxmerkt** [comment]
Thank you @CameronDevine

**Comment by CameronDevine** [comment]
@wxmerkt, I am unsure of which files should be installed. I know the files in the [launch](https://github.com/dmalyuta/apriltags2_ros/tree/master/apriltags2_ros/launch) and [config](https://github.com/dmalyuta/apriltags2_ros/tree/master/apriltags2_ros/config) directories should be installed, but I am unsure about the files in the [scripts](https://github.com/dmalyuta/apriltags2_ros/tree/master/apriltags2_ros/scripts) directory. I believe [analyze_image.sh](https://github.com/dmalyuta/apriltags2_ros/blob/master/apriltags2_ros/scripts/analyze_image.sh) could be run with `rosrun`, but [calibrate_bundle.m](https://github.com/dmalyuta/apriltags2_ros/blob/master/apriltags2_ros/scripts/calibrate_bundle.m) could not because it is a Matlab script. Should either of these two files be installed?

**Comment by wxmerkt** [comment]
I don't think so. The `analyze_image.sh` is a convenience wrapper around another roslaunch call so I'd rather not include it.

**Comment by CameronDevine** [comment]
I submitted #42 tonight which fixes this issue for me.

**Comment by wxmerkt** [comment]
Thank you for the pull request. I have incorporated along many others into a full upgrade of the library. Please try #43 whether this fixes your issue.

**Comment by wxmerkt** [comment]
This should have been fixed by #43.
